src/db/repository/postgres/AgenciesRepository.ts

```diff
@@ -0,0 +1,64 @@
+import { Kysely, UpdateObject } from 'kysely';
+
+import { Agencies, DB } from '../../types/generated/database';
+
+export type AgenciesUpdateParams = Omit<
+	UpdateObject<DB, 'agencies'>,
+	'id' | 'created_at' | 'updated_at'
+>;
+export type AgenciesCreateParams = Omit<
+	Agencies,
+	'id' | 'is_verified' | 'created_at' | 'updated_at' | 'deleted_at'
+>;
+
+export default class AgenciesRepository {
+	constructor(private readonly database: Kysely<DB>) {}
+
```

Comment:
```suggestion

	getById(id: string) {
		return this.database
			.selectFrom('agencies')
			.where('id', '=', id)
			.executeTakeFirstOrThrow();
	}

```

Added a `getById` method for parity with other repositories.